### ISSUE TITLE
Auto expire events list

### ISSUE THREAD

**Issue by johnsimons** [Particular/ServicePulse#182]
Old messages stay a long time in the main window. When the message is taking care for and it’s finished, I don’t want see it a month or longer in the main screen. When you add a archive field in the database for the message, the dba can decide to archive the messages. So i miss a button to archive and/or remove the messages.
![image](https://cloud.githubusercontent.com/assets/3889023/3872943/89edc504-2126-11e4-9bae-ba7c71204867.png)

**Comment by WojcikMike** [comment]
I agree that sticking items on main screen doesn't make sense. For me this list should only contain things that one haven't handle/acknowledge. So I agree with a archive/remove button approach.
